frontend/components/notifications/SubscriptionDialog.vue

```diff
@@ -0,0 +1,232 @@
+      <NotificationsSubscriptionRow v-model="modifiableOptions.syncCommittee" :t-path="tPath+'sync_committee'" :lacks-premium-subscription="!isOptionAvailable('syncCommittee')" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.withdrawn" :t-path="tPath+'withdrawn'" :lacks-premium-subscription="!isOptionAvailable('withdrawn')" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.slashed" :t-path="tPath+'slashed'" :lacks-premium-subscription="!isOptionAvailable('slashed')" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.realTime" :t-path="tPath+'real_time'" :lacks-premium-subscription="!isOptionAvailable('realTime')" class="row" />
+      <hr>
+      <NotificationsSubscriptionRow v-model="allCheckbox" :t-path="tPath+'all'" :lacks-premium-subscription="false" class="row" />
+    </div>
+
+    <div v-else-if="props.accountSub">
+      <NotificationsSubscriptionRow v-model="modifiableOptions.incoming" :t-path="tPath+'incoming'" :lacks-premium-subscription="!isOptionAvailable('incoming')" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.outgoing" :t-path="tPath+'outgoing'" :lacks-premium-subscription="!isOptionAvailable('outgoing')" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.erc20" :t-path="tPath+'erc20'" :lacks-premium-subscription="!isOptionAvailable('erc20')" input-type="amount" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.erc721" :t-path="tPath+'erc721'" :lacks-premium-subscription="!isOptionAvailable('erc721')" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.erc1155" :t-path="tPath+'erc1155'" :lacks-premium-subscription="!isOptionAvailable('erc1155')" class="row" />
+      <hr>
+      <NotificationsSubscriptionRow v-model="allCheckbox" :t-path="tPath+'all'" :lacks-premium-subscription="false" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.networks" :t-path="tPath+'networks'" :lacks-premium-subscription="!isOptionAvailable('networks')" input-type="networks" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.ignoreSpam" :t-path="tPath+'ignore_spam'" :lacks-premium-subscription="!isOptionAvailable('ignoreSpam')" class="row" />
+    </div>
+
+    <div class="footer">
+      <Button type="button" :label="t('notifications.subscriptions.button')" @click="closeDialog" />
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+@use '~/assets/css/fonts.scss';
+
+.content {
+  display: flex;
+  flex-direction: column;
+
+  .title {
+    @include fonts.dialog_header;
+    text-align: center;
+    margin-bottom: var(--padding-large);
+  }
+
+  .explanation {
+    margin-bottom: var(--padding);
+    @include fonts.small_text;
+    opacity: 0.6;
```

Comment:
OK, I set it to ` --text-color-discreet`.
I had used opacity because the discreet text looks almost black with the light color theme. I do not see the point then.

I contacted Philip to check that it is really what we want.